### ISSUE TITLE
Auto delete expired links

### ISSUE THREAD

**Issue by jdevalk** [lesterchan/wp-draftsforfriends#4]
Would be cool if the plugin would automatically delete expired links after 48h or so.

**Comment by lesterchan** [comment]
That is not bad an idea!

**Comment by salim-b** [comment]
I totally agree with this idea! But please make this optional and include the possibility to set a custom delay value for auto-deletion – preferably in the same units as the expiration timer is set now (secs, mins, hours and days).

Additionally an option to automatically delete expired links _**after the corresponding post is published**_ would be awesome! :metal:
